Add callout for sendEventsOnFlagRead (#320)
launchdarkly/LaunchDarkly-Docs: src/content/topics/sdk/client-side/react/react-web.mdx

Versions 2.26 and earlier of the React SDK send an evaluation event when a user reads a flag evaluation using `allFlags` and `variation`. To recreate this behavior in version 2.27 and later, explicitly set `options.sendEventsOnlyForVariation: false` and set `reactOptions.sendEventsOnFlagRead: false`. Version 3 only sends events when a context receives a flag evaluation.

<Callout intent="alert">
<CalloutTitle>Experimentation events</CalloutTitle>
<CalloutDescription>

> Setting `reactOptions.sendEventsOnFlagRead: false` could prevent experimentation events from recording correctly. Always check to see if this option is disabled when troubleshooting experiments.

Reviewer: ```suggestion
Setting `reactOptions.sendEventsOnFlagRead: false` could prevent experimentation events from recording correctly. Always confirm that this option is disabled when troubleshooting experiments.
```
